commands/run.sh

```diff
@@ -85,7 +88,7 @@ EOF
     cap_run_dry_run
   else
     sbatch -D "$job_directory" \
-      --job-name="$CAP_PROJECT_NAME\_${job_name%.*}" \
+      --job-name="$CAP_PROJECT_NAME-${job_name%.*}" \
```

Comment:
After digging around a little bit more and trying a few more things I noticed that the pipeline.sh file still has `LAB` instead of `CAP` for both the project name and the random seed. Once I switch `LAB_PROJECT_NAME` to `CAP_PROJECT_NAME`, my jobs in the queue will have the correct name. I am not really sure why those still populate with `LAB` and not `CAP`.

Reply:
Finally, having the job name first would be more helpful so users can see which script they are running when they look at their queue on the command line instead of the project. Since the project name comes before the script name and the job name becomes fairly long, you only see the project name in this small/quick view.

Reply:
@tsoelter Thanks for finding that!!  I had made the change and forgot to push it to Github.  I have pushed it and merged it into the project-template repository.  Please rerun the setup commands in the pull request description, create a new test project, and verify it works correctly.

Ok, I will switch the names.

Reply:
@toniecrumley Looks like everything works as it should now, and I like the swapped job name! 🎉  I had to pull an additional time during the setup to get the `pipeline.sh` change, which was not in the original pull request description (see below what I had to do):
```
cd ~/bin/capture
git checkout main
git pull
git checkout cap-data-download
git pull
git submodule update --init --recursive
```